README.md

```diff
@@ -73,8 +73,9 @@ Start with the [README](docs/README.md) file in the docs directory
 
```

Comment:
This comment is a little out of the scope of the changes in this CR.  I've been using nvm and by using nvm I can install multiple versions of node and change to whichever version a project might need.  I can also set the default node version to any version.  Using nvm I don't need to maintain symbolic links.

Reply:
Good thing to note - but, yeah, let's call that out-of-scope for issues addressed in this PR.

If you want to create a new feature issue "Node Best Practices", that'd be fine. If you also want to work on implementing it, You could eitehr make it a sibling to the other "Development Best Practices" sections, or add it to the "Javascript Best Practices" section: https://uazgraduatecollege.github.io/developer-essentials/#/best-practices/javascript

But for now, please go ahead and mark this PR as reviewed.